pkg/orchestrator/scheduler/ops_job.go

```diff
@@ -120,7 +120,7 @@ func (b *OpsJobScheduler) Process(ctx context.Context, evaluation *models.Evalua
 func (b *OpsJobScheduler) createMissingExecs(
 	ctx context.Context, job *models.Job, plan *models.Plan) (execSet, error) {
 	newExecs := execSet{}
-	nodes, err := b.selector.AllMatchingNodes(ctx, job)
```

Comment:
In the event a user submits an `ops` job requiring 5 CPU and the nodes currently present in the network support 4 CPU we mark the job as failed, rather than queue it - is that right? Or if the job requires 1 GPU and none of the nodes have a GPU we also fail it?

Reply:
I am not sure of we mark it as failed or just mark it as completed with 0 executions/node (currently on my phone)
But the expected behaviour for ops jobs to run on the _current_ nodes that match the job requirements. It shouldn't queue and shouldn't schedule the job on newly joined nodes after the job has started. I would hope to fail if 0 nodes were matched

Reply:
Currently we fail if zero nodes match.

Reply:
Nice. Thats how it should be